packages/react/src/use-render/useRender.ts

```diff
@@ -63,9 +52,9 @@ export namespace useRender {
      */
     render: RenderProp<State>;
     /**
-     * Refs to be merged together to access the rendered DOM element.
+     * The ref to apply to the rendered element.
      */
-    refs?: React.Ref<RenderedElementType>[];
+    ref?: React.Ref<RenderedElementType> | React.Ref<RenderedElementType>[];
```

Comment:
I standardized this with `useRenderElement`, both use `ref` for the propname, while still allowing an array.